db/people.yml

```diff
@@ -414,10 +414,37 @@ btabanpour:
   position: masters research assistant
 cbracy:
   active: true
-  aka: []
-  avatar: https://github.com/.png
-  bio: bio
-  education: []
+  aka:
+    - Connor Bracy
+    - Connor J. Bracy
+    - Bracy, Connor
+    - Bracy, Connor J.
+    - Bracy, C. J.
+  avatar: https://avatars.githubusercontent.com/u/7377785
+  bio: Connor was born and raised in a suburb just outside of Seattle Washington
+       where he continued to live until coming to Columbia. Having originally
+       started his higher education pursuing programming physics engines, he
+       eventually transitioned to pursuing physics research. When he’s not
+       working, you might find him running, rock climbing, skateboarding, or
+       just having a night in playing video games.
+  education:
+    - begin_year: 2014
+      begin_month: 9
+      degree: B.S. in Computer Science
+      institution: DigiPen Institute of Technology
+      end_year: 2017
+      end_month: 8
+    - begin_year: 2016
+      begin_month: 6
+      degree: B.S. in Physics
+      institution: University of Washington
```

Comment:
this is fine actually, but since you are editing, we like to put an id here and dereference from institutions, so this would be `uwashington`